cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectApiImpl.java

```diff
@@ -443,14 +443,9 @@ public Future<Map<String, String>> listConnectLoggers(Reconciliation reconciliat
                                             Map<String, Map<String, String>> fetchedLoggers = mapper.readValue(buffer.getBytes(), MAP_OF_MAP_OF_STRINGS);
                                             Map<String, String> loggerMap = new HashMap<>(fetchedLoggers.size());
                                             for (var e : fetchedLoggers.entrySet()) {
```

Comment:
while we're here we might as well change `e` to something a bit more meaningful, like `loggerEntry`

Reply:
Sure, changed to this.